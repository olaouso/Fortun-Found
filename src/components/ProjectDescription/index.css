.projectDiscription {
  margin: 2em;
}

.projectDescriptionRow {
  display: flex;
  align-items: center;
  justify-content: center;
  text-align: center;
}

.projectDescriptionHeader {
  font-size: 1.75em;
  color: #03c4a1;
}

.projectDescriptionHeaderAndParagraph {
  width: 100%;
  padding-right: 3em;
  font-family: "Acme";
  text-align: center;
}

.projectDescriptionParagraph {
  padding-top: 1em;
  /* margin-top: 0em; */
  font-size: 1.25em;
  color: #726a95;
}

.projectDescritionkImg {
  width: 70%;
  padding: 1%;
  margin: 1%;
  text-align: center;
  /* margin: 1.25em;
  margin-left: 0em;  */
}
