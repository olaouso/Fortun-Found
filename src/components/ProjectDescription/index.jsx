import React from "react";
import "./index.css";
import { Row, Col } from "react-bootstrap";
import { useTranslation } from "react-i18next";
import giftImg from "./giftImg.jpg";

export default function ProjectDescription() {
  const { t } = useTranslation();

  return (
    <div className="projectDiscription" fluid="md">
      <Row className="projectDescriptionRow">
        <Col md={6}>
          <img
            src={giftImg}
            alt="Team work"
            className="projectDescritionkImg"
          />
        </Col>
        <Col md={6}>
          <div className="projectDescriptionHeaderAndParagraph">
            <h1 className="projectDescriptionHeader">
              {t("projectDescription.header")}
            </h1>
            <p className="projectDescriptionParagraph">
              {t("projectDescription.paragraph")}
            </p>
          </div>
        </Col>
      </Row>
    </div>
  );
}
