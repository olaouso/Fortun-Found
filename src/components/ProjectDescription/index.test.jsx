import React from "react";
import renderer from "react-test-renderer";
import ProjectDescription from "./index";

it("renders ProjectDescription correctly", () => {
  const tree = renderer.create(<ProjectDescription />).toJSON();
  expect(tree).toMatchSnapshot();
});
