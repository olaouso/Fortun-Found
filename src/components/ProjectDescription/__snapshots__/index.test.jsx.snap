// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ProjectDescription correctly 1`] = `
<div
  className="projectDiscription"
  fluid="md"
>
  <div
    className="projectDescriptionRow row"
  >
    <div
      className="col-md-6"
    >
      <img
        alt="Team work"
        className="projectDescritionkImg"
        src="giftImg.jpg"
      />
    </div>
    <div
      className="col-md-6"
    >
      <div
        className="projectDescriptionHeaderAndParagraph"
      >
        <h1
          className="projectDescriptionHeader"
        >
          projectDescription.header
        </h1>
        <p
          className="projectDescriptionParagraph"
        >
          projectDescription.paragraph
        </p>
      </div>
    </div>
  </div>
</div>
`;
